espnet2/asr/frontend/sinc.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+# encoding: utf-8
```

Comment:
`encoding: utf-8` is meaning less in python3, and it's not recommended by PEP

Reply:
I removed those lines.